editoast/src/main.rs

```diff
@@ -60,6 +62,26 @@ async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
     }
 }
 
+fn init_sentry(args: &RunserverArgs) -> Option<ClientInitGuard> {
+    match (args.sentry_dsn.clone(), args.sentry_env.clone()) {
+        (Some(sentry_dsn), Some(sentry_env)) => Some(sentry::init((
+            sentry_dsn,
+            sentry::ClientOptions {
+                release: match env::var("OSRD_GIT_DESCRIBE").ok() {
+                    Some(release) => Some(release.into()),
+                    None => sentry::release_name!(),
+                },
+                environment: Some(sentry_env.into()),
+                ..Default::default()
+            },
+        ))),
+        _ => {
+            println!("SENTRY_DSN and SENTRY_ENV must be set to send events to Sentry.");
+            None
```

Comment:
If both SENTRY_DSN and SENTRY_ENV aren't set, I think it's safe to say the user didn't intend to use sentry, and thus that no warning should be displayed